### ISSUE TITLE
The example does not deploy

### ISSUE THREAD

**Issue by juan-acevedo-ntt** [claranet/terraform-azurerm-app-gateway#9]
Hi!

Even I corrected several errors in the example (see https://github.com/claranet/terraform-azurerm-app-gateway/pull/8) I still get an error when trying to deploy it:

```
│ Error: creating Application Gateway: (Name "agw-testjja-ntt-euw-dev" / Resource Group "rg-testjja-ntt-dev"): network.ApplicationGatewaysClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="InvalidResourceReference" Message="Resource /subscriptions/1341fec5-a980-41e9-a3b8-e99f509917aa/resourceGroups/rg-testjja-ntt-dev/providers/Microsoft.Network/applicationGateways/agw-testjja-ntt-euw-dev/frontendIPConfigurations/testjja-ntt-euw-dev-frontipconfig referenced by resource /subscriptions/1341fec5-a980-41e9-a3b8-e99f509917aa/resourceGroups/rg-testjja-ntt-dev/providers/Microsoft.Network/applicationGateways/agw-testjja-ntt-euw-dev/httpListeners/testjja-ntt-euw-dev-listener-https was not found. Please make sure that the referenced resource exists, and that both resources are in the same region." Details=[]
│ 
│   with module.appgw_v2.azurerm_application_gateway.app_gateway,
│   on .terraform/modules/appgw_v2/r-application-gateway.tf line 1, in resource "azurerm_application_gateway" "app_gateway":
│    1: resource "azurerm_application_gateway" "app_gateway" {
│ 
``` 

I am using Terraform v1.3.2

**Comment by Shr3ps** [comment]
Hi,

We will check the example sample code.

Also, please note that  for the next major version (v7) of this module all variables will be typed.

**Comment by Shr3ps** [comment]
Fixed by the latest release `v7.0.1`
